Sources/NIOHPACK/HPACKHeader.swift

```diff
@@ -394,6 +394,9 @@ extension HPACKHeaders: CustomStringConvertible {
     }
 }
 
+// NOTE: This is a bad definition of equatable and hashable. In particular, both order and
+// indexability are ignored. We're sort of stuck with this behaviour now and cannot change it,
+// but in the next major version we should strongly consider revising it.
```

Comment:
Should we link a GH issue?

Reply:
Oh, I think this comment is just unnecessarily aggressive.